### DISCUSSION TITLE
How to connect to MySQL on Planetscale (which requires SSL connection)?

### DISCUSSION THREAD

**Discussion by neoromantic** [Budibase/budibase#4306] [Help]
Hey there.

I'm new to Budibase, but I definitely looking forward to use it our products.

Can't connect to Mysql instance on Planetscale, because of SSL.

Can't find any docs on this on Budibase and in Github.

Could please somebody explain how can I connect with SSL to my MySQL db?

**Comment by mjashanks** [comment]
Hi @neoromantic  - welcome.

When using SSL, click the "Add" button to set the options of your SLL connection:

![image](https://user-images.githubusercontent.com/3524181/152328108-da113fb0-a9a2-42c5-b664-ea9a60579524.png)

The "Key" and "Value" refers to these options https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options

What options you use will depend on what's required/supported by PlanetScale

**Comment by neoromantic** [comment]
Thank you for answer, but I still don't understand. Here's what PlanetScale documentation suggests for connection with CLI:

```mysql --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/certs/ca-certificates.crt```

I don't understand how to specify similar things for Budibase. 

Which certificate should I use for Budibase? I've tried those: 
- /etc/ssl/certs/ca-certificates.crt
- /etc/pki/tls/certs/ca-bundle.crt

both led to 500 error "error loading local certificate".

**Reply by RASS00** [reply]
hey! did you find a solution?

**Reply by wmlutz** [reply]
Checking in 2023, anyone get a good guide going for how to connect budibase to planetscale?

**Reply by multiplehats** [reply]
In Planetscale click on "Get connection strings"

Select Laravel, then you'll see:

```
MYSQL_ATTR_SSL_CA=/etc/ssl/certs/ca-certificates.crt
```

Use `MYSQL_ATTR_SSL_CA` as the key and `/etc/ssl/certs/ca-certificates.crt` as the value
